test/fuzz/basic.ml

```diff
@@ -514,6 +514,56 @@ let data_drop (env : Env.t) =
     (fun name -> pair (const (Data_drop (Symbolic name))) (const [ S.Nothing ]))
     env.datas
 
+let elem_drop (env : Env.t) =
+  List.map
+    (fun (name, _) -> pair (const (Elem_drop (Symbolic name))) (const [ S.Nothing ]))
+    env.elems
+
+let table_init (env : Env.t) =
+  List.flatten @@
+  List.map
+    (fun (name_x, _) -> 
+      List.map (fun (name_y, _) -> pair (const (Table_init (Symbolic name_x, Symbolic name_y) )) (const [ S.Pop; S.Pop; S.Pop ]))
+      env.tables
+    )
+    env.tables
+(* TODO: to be continued > add constraint between table $name_x and $name_y *)
+
+let table_copy (env : Env.t) =
+  List.flatten @@
+  List.map
+    (fun (name_x, _) -> 
+      List.map (fun (name_y, _) -> pair (const (Table_copy (Symbolic name_x, Symbolic name_y) )) (const [ S.Pop; S.Pop; S.Pop ]))
+      env.tables
+    )
+    env.tables
```

Comment:
Let's rewrite this in another way.

What about something like:

```ocaml
let tables = List.map const env.tables in
let* t1 = choose tables in
let* t2 = choose tables in
pair (
  const (
    Table_copy (
      Symbolic t1,
      Symbolic t2)))
(const [S.Pop; S.Pop; S.Pop])
```

(untested :-))

Reply:
Is it to avoid `List.flatten` (due to nested `List.map`) ?

Reply:
Type of this instruction is `(instr * S.stack_op list) gen` but it's expected `(instr * S.stack_op list) gen list`

Reply:
Done : https://github.com/OCamlPro/owi/pull/28/commits/4b296e6fea835931179ca46856dbd243f15c5534
In `table_copy` function, is it possible to avoid `if rt_x = rt_y then ... else pair (const (Nop)) (const [ S.Nothing ])` ?
I try `option ...` without success

Reply:
I believe this is fine for now. Could you simply add a `TODO` explaining that this should be removed at some point ? :)